ext/dba/dba.stub.php

```diff
@@ -2,11 +2,21 @@
 
 /** @generate-class-entries */
 
+#ifdef DBA_LMDB
+/** @var int */
+const DBA_LMDB_USE_SUB_DIR = 0;
```

Comment:
This is where the constant is defined

Reply:
No, the error message mentions the other one :D 

> Constant DBA_LMDB_NO_SUB_DIR

Reply:
I'm an idiot...